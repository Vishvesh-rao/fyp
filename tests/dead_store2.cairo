func are_equal(x, y) -> (eq):
    if x == y:
        let (a) = 44
        return (1)
    else:
        return (0)
    end
end
