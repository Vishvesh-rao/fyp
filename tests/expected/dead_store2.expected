[dead-store] in dead_store2.cairo:3:14